Lib/idlelib/util.py

```diff
@@ -0,0 +1,36 @@
+"""
+Idlelib objects with no external idlelib dependencies
+which are needed in more than one idlelib module.
+
+They are included here because
+    a) they don't particularly belong elsewhere; or
+    b) because inclusion here simplifies the idlelib dependency graph.
+
+TODO:
+    * Python versions (editor and help_about),
+    * tk version and patchlevel (pyshell, help_about, maxos?, editor?),
+    * std streams (pyshell, run),
+    * warning stuff (ditto).
```

Comment:
```suggestion
    * warning stuff (pyshell, run).
```
Make it explicit, so this line doesn't need to be changed when the point above is completed.